src/docker-osx-dev

```diff
@@ -562,7 +554,7 @@ function install_dependencies {
   brew update
 
   brew_install "caskroom/cask/brew-cask" "Cask" "" false
-  brew_install "virtualbox" "VirtualBox" "VBoxManage" true
+  brew_install "virtualbox" "VirtualBox" true
```

Comment:
The `brew_install` function expects 4 arguments. With three, I think it will end up looking for a command line app called `true`, which does exist, so I suppose this works, but it's confusing for a reader of the code.